pypeit/par/util.py

```diff
@@ -565,6 +565,7 @@ def make_pypeit_file(pypeit_file, spectrograph, data_files, cfg_lines=None, setu
     else:
         _cfg_lines = list(cfg_lines)
 
+    # TODO: Can we get rid of these lines?
```

Comment:
yes, we can and should.

Reply:
Done.  Added a TODO to remind us that we might want to bring back checks for the number of calibration frames.